pom.xml

```diff
@@ -308,7 +308,7 @@ under the License.
 				errors.
 
 				[1] https://github.com/netty/netty/issues/3704 -->
```

Comment:
Looks like you can remove this comment now.

Reply:
Ops, dropped.